The Travis build is failing with the following error:
```
./integration_test.go:17:2: undefined: testing.Init
```
Is the testing fix 1.13+ specific? If so we should find a way to be more inclusive down to 1.11 in order to be able to contribute this upstream.

--- a/integration_test.go
+++ b/integration_test.go
@@ -13,11 +13,6 @@ var vaultRoleID, vaultSecretID, noKVRoleID, noKVSecretID string
 
 var vaultVersion string
 
-var _ = func() bool {
-	testing.Init()
-	return true
-}()
-
 func init() {
 	flag.StringVar(&vaultVersion, "vaultVersion", "1.0.1", "provide vault version to be tested against")
 	flag.Parse()

--- a/test-files/initVaultDev.sh
+++ b/test-files/initVaultDev.sh
@@ -5,21 +5,21 @@ export VAULT_VERSION=${1:-1.0.1}
 
 case "$(uname -s)" in
   Darwin*)
-    os="darwin_amd64"
+    OS="darwin_amd64"
     ;;
   MINGW64*)
-    os="windows_amd64"
+    OS="windows_amd64"
     ;;
   *)
-    os="linux_amd64"
+    OS="linux_amd64"
     ;;
 esac
 
 CURRENT_VAULT=`./vault version | cut -d'v' -f2 | cut -d' ' -f1`
 if [ "$CURRENT_VAULT" != "$VAULT_VERSION" ]; then
     rm -rf ./vault
-    curl -kO https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_${os}.zip
-    unzip vault_${VAULT_VERSION}_${os}.zip
+    curl -kO https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_${OS}.zip
+    unzip vault_${VAULT_VERSION}_${OS}.zip
 fi
 
 ./vault server -dev -dev-root-token-id ${VAULT_TOKEN}  > /tmp/vaultdev.log &

